content/en/docs/Tracing/tempo-walkthrough.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Tracing with Grafana Tempo
+description: Tracing with Grafana Tempo.
+weight: 10
+---
+
+## Ingress NGINX Setup
+
+In order to access Grafana, we need an ingress controller, we can install `ingress-nginx` with the following command:
```

Comment:
Is this only supposed to work on KinD? This doesn't come up for me on K3d.

Reply:
Need to state this is specifically for KinD and assumes the user's system does not already have something binding to ports 80 and 443.

Reply:
Done